src/components/action-buttons/action-buttons.component.tsx

```diff
@@ -53,6 +75,19 @@ function ActionButtons({ schema, t }: ActionButtonsProps) {
     }
   }
 
+  async function handleValidateAndPublish() {
+    setStatus('validateBeforePublishing');
+    const [errorsArray] = await handleFormValidation(schema, dataTypeToRenderingMap);
+    setValidationResponse(errorsArray);
+    if (errorsArray.length > 0) {
+      setStatus('validated');
+      setValidationComplete(true);
+      setPublishedWithErrors(true);
```

Comment:
Does this mean we are going to let the user publish the form if it has validation errors?

Reply:
No actually, it means that set a state that the user is trying to publish with errors, I need that state to show some inlineNotification.